services/src/main/java/io/grpc/services/AdminInterface.java

```diff
@@ -19,6 +19,7 @@
 import io.grpc.BindableService;
 import io.grpc.ExperimentalApi;
 import io.grpc.ServerServiceDefinition;
+import io.grpc.protobuf.services.ChannelzService;
```

Comment:
This dependency should be ok. Or we can change it when making io.grpc.protobuf.services a separate artifact.